templates/slim/block_image.html.slim

```diff
@@ -1,10 +1,11 @@
-.imageblock(id=@id class=[@style,role]
-    style=[("text-align: #{attr :align}" if attr? :align),("float: #{attr :float}" if attr? :float)].compact.join('; '))
-  .content
-    - if attr? :link
-      a.image href=(attr :link)
+- unless attr(1) == 'background' || attr(1) == "canvas"
+  .imageblock(id=@id class=[@style,role]
```

Comment:
I have to admit, I don't know what `@style` does and where it comes from... I just copied and pasted stuff around. I'm a sham. ;)